aws_lambda_builders/workflow.py

```diff
@@ -61,7 +61,7 @@ def wrapper(self, *args, **kwargs):
         if len(self.binaries) != len(valid_paths):
             raise WorkflowFailedError(workflow_name=self.NAME,
                                       action_name=None,
-                                      reason='Binary validation failed!')
+                                      reason='Unable to find a valid executable for {}'.format(self.runtime))
```

Comment:
This isnt necessarily the runtime, it could be the package manager, gradle, maven as well.

Reply:
Hmm.. Is there a better message we can use?

Reply:
well self.binaries could be used instead of self.runtime I think, it has total list of binaries that are needed to be verified.

Reply:
@sanathkr @TheSriram Did we ever land on what this should be? Would be great to wrap this up and get it into an upcoming release.